ncats_webd/cybex_queries.py

```diff
@@ -278,6 +279,7 @@ def csv_get_closed_tickets(db, ticket_severity):
             index=False,
             date_format="%Y-%m-%d %H:%M:%S",
             columns=[
+                "id",
```

Comment:
Use correct field name:
```suggestion
                "_id",
```

Reply:
updated